apps/i18n/common/da_dk.json

```diff
@@ -1285,7 +1286,7 @@
   "teacherTabAssessments": "",
   "teacherTabManageStudents": "Administrer elever",
   "teacherTabProgress": "Fremskridt",
-  "teacherTabProjects": "Projekter",
+  "teacherTabProjects": "",
```

Comment:
@Hamms I noticed that an empty string is returned for "teacherTabProjects" in several languages.  Is this possibly due to the original string in English being changed?  I checked to see if there is a diff in en_us.json file, but there is no change.  Thanks.